cli/flox-activations/src/cli/fix_paths.rs

```diff
@@ -0,0 +1,216 @@
+    let path_dirs = separate_dir_list(path_var);
+    let flox_env_dirs = separate_dir_list(flox_env_dirs_var);
+    let suffixes = ["bin", "sbin"];
+    let fixed_path_dirs = prepend_dirs_to_pathlike_var(&flox_env_dirs, &suffixes, &path_dirs);
+    join_dir_list(fixed_path_dirs)
+}
+
+/// Calculate the new man(1) search path from FLOX_ENV_DIRS and the existing MANPATH.
+///
+/// Note that we *always* add a trailing ':' to MANPATH because the search path
+/// for man pages is complicated.
+///
+/// Depending on whether leading/trailing ':' characters are present in MANPATH,
+/// you get different behavior. Let "STD" be the standard search path.
+/// ex.) (leading) MANPATH=":foo" -> search path is "STD:foo"
+/// ex.) (trailing) MANPATH="foo:" -> search path is "foo:STD"
+/// ex.) (two consecutive) MANPATH="foo::bar" -> search path is "foo:STD:bar"
+/// ex.) (not leading or trailing) MANPATH="foo" -> search path is "foo"
+///
+/// So, we always put a trailing ':' in MANPATH so that man pages from the
+/// active environments take precedence without *removing* the standard
+/// search path.
+fn fix_manpath_var(flox_env_dirs_var: &str, manpath_var: &str) -> String {
+    let manpath_dirs = separate_dir_list(manpath_var);
+    let flox_env_dirs = separate_dir_list(flox_env_dirs_var);
+    let suffixes = ["share/man"];
+    let fixed_manpath_dirs = prepend_dirs_to_pathlike_var(&flox_env_dirs, &suffixes, &manpath_dirs);
+    let mut joined = join_dir_list(fixed_manpath_dirs);
+    if !joined.ends_with(':') {
+        joined.push(':');
+    }
+    joined
+}
+
+#[cfg(test)]
+mod test {
+    use std::str::FromStr;
+
+    use super::*;
+
+    #[test]
+    fn appends_suffixes() {
+        let suffixes = ["suffix"];
+        let flox_env_dirs = [PathBuf::from_str("/flox_env").unwrap()];
+        let path_dirs = [
+            PathBuf::from_str("/path1").unwrap(),
+            PathBuf::from_str("/path2").unwrap(),
+        ];
+        let new_dirs = prepend_dirs_to_pathlike_var(&flox_env_dirs, &suffixes, &path_dirs);
+        assert_eq!(new_dirs[0], PathBuf::from_str("/flox_env/suffix").unwrap());
+    }
+
+    #[test]
+    fn flox_envs_added_in_order() {
+        let suffixes = ["suffix"];
+        let flox_env_dirs = [
+            PathBuf::from_str("/flox_env1").unwrap(),
+            PathBuf::from_str("/flox_env2").unwrap(),
+        ];
+        let path_dirs = [
+            PathBuf::from_str("/path1").unwrap(),
+            PathBuf::from_str("/path2").unwrap(),
+        ];
+        let new_dirs = prepend_dirs_to_pathlike_var(&flox_env_dirs, &suffixes, &path_dirs);
+        assert_eq!(new_dirs, vec![
+            PathBuf::from_str("/flox_env1/suffix").unwrap(),
+            PathBuf::from_str("/flox_env2/suffix").unwrap(),
+            PathBuf::from_str("/path1").unwrap(),
+            PathBuf::from_str("/path2").unwrap(),
+        ]);
+    }
+
+    #[test]
+    fn duplicate_paths_removed() {
```

Comment:
nonblocking: looks like it's consistent with existing behavior, but I don't think we need to remove duplicates of paths users have put in PATH

Reply:
Yeah I'm just trying to be consistent with existing behavior. Happy to submit a followup if we decide that's important.